core/src/main/java/edu/wpi/grip/core/operations/Operations.java

```diff
@@ -15,6 +15,7 @@ private Operations() { /* no op */}
 
     public static void addOperations(EventBus eventBus) {
         // Add the default built-in operations to the palette
+        eventBus.post(new OperationAddedEvent(new ResizeOperation()));
```

Comment:
We don't have a sort order the order is defined by the order that we list them in this file and the generated operations file.